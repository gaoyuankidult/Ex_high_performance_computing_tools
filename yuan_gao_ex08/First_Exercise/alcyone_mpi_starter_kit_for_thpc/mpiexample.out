./submit_parallel: line 105: srun: command not found
