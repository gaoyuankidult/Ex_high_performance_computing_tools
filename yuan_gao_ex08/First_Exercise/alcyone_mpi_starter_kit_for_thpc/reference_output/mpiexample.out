
host alm01

host alm01

host alm01

host alm01

host alm01

host alm01

host alm01
message: 6 12 sender: 6
message: 3 12 sender: 3
message: 2 12 sender: 2
message: 1 12 sender: 1
message: 11 12 sender: 11
message: 4 12 sender: 4
message: 9 12 sender: 9
message: 8 12 sender: 8
message: 10 12 sender: 10
message: 7 12 sender: 7
message: 5 12 sender: 5

host alm01

host alm01

host alm01

host alm01

host alm01
